src/Sylius/Component/Rbac/spec/Model/Permission.php

```diff
@@ -0,0 +1,78 @@
+<?php
```

Comment:
Wrong File name? It doesn't end in Spec. Not sure but with old phpspec versions it wasn't parsed.